regression/goto-diff/java-mod-identifier2/new/Test.java

```diff
@@ -0,0 +1,18 @@
+public class Test {
+  public int foo(int x) {
+    if (x > 10) {
+      return x;
+    } else {
+      int y = x * 10;
```

Comment:
This has done more than modify an identifier, as it adds an extra statement too. I suggest adding a test like:

```
int f(int x, int y) { return x; }
->
int f(int x, int y) { return y; }
```

That will test whether something that *looks* like a renaming but actually makes a substantial change is detected.

Reply:
Thanks @smowton I have fixed this in the most recent upload. @peterschrammel if you are still working on this branch I have done a force push to put the fix in the original commit.